src/main/resources/application.yaml

```
@@ -0,0 +1,64 @@
+quarkus:
+  http:
+    auth:
+      basic: true
+      policy:
+        main-policy:
+          roles-allowed: admin
+      permission:
+        main:
+          paths: /*
+          policy: main-policy
+        public:
+          paths: /q/health/*
+          policy: permit
+          methods: GET
+  security:
+    users:
+      file:
+        realm-name: pim_file
+        enabled: true
+        plain-text: true
+        users: users.properties
+        roles: roles.properties
+    jdbc:
+      realm-name: Quarkus_jdbc
+      enabled: true
```

shouldn't this be disabled, as per default realm it's by property files?
```suggestion
      enabled: false
```

---

this was added to enable runtime possibility to configure authentication. This property is a build time property